ci/jenkins/pipelines/skuba-e2e-test.Jenkinsfile

```diff
@@ -31,7 +35,9 @@ node('caasp-team-private-integration') {
 
         if (env.REPO_BRANCH != ""){
                branch_repo = "http://download.suse.de/ibs/Devel:/CaaSP:/5:/Branches:/${env.REPO_BRANCH}/SLE_15_SP2"
-        }
+               branch_registry = "registry.suse.de/devel/caasp/5/branches/${env.REPO_BRANCH}/containers/cr/containers/caasp/v5"
```

Comment:
this is wrong, see the cilium example to check the registry URI. It should be 

`registry.suse.de/devel/caasp/5/branches/cilium17/containers/caasp/v5`

The schema of this url is as follows:

`<host>/<project_path>/<project_repo>/<image_name>:<image_tag>`

I believe that the following should also work and this way we do not include any specific image value inside the configuration (even I doubt we ever will customize the  `caasp/v5` prefix in some of our images but not all):

 `branch_registry = "registry.suse.de/devel/caasp/5/branches/cilium17/containers"`
and
`original_registry ="registry.suse.de/devel/caasp/5/containers/cr/containers"`